extra/modules/greenbids-real-time-data/src/main/java/org/prebid/server/hooks/modules/greenbids/real/time/data/model/data/Partner.java

```diff
@@ -11,6 +11,9 @@
 @Value(staticConstructor = "of")
 public class Partner {
 
+    @JsonProperty(required = true)
+    Boolean enabled;
```

Comment:
not sure you need this particular flag, because it's possible to enable/disable the module using execution plan or hooks.admin.module-execution properties on the account or host-level config and the core skips execution of the module for you